src/Google/Ads/GoogleAds/V2/Services/resources/mutate_job_service_rest_client_config.php

```diff
@@ -62,64 +62,5 @@
                 ],
             ],
         ],
-        'google.longrunning.Operations' => [
```

Comment:
Would this affect mutate job service? It's currently using longrunning.
Could you follow up after the release?

Reply:
Good catch, I will follow up on this.